staff/dtiburcio/JS-Exercises/JavascriptExercises.js

```diff
@@ -0,0 +1,177 @@
+/*
+translate()
+Write a function translate() that will translate a text into "rövarspråket". That is, double every consonant and place an occurrence of "o" in between. For example, translate("this is fun") should return the string "tothohisos isos fofunon".
+*/
+
+function translateText(textForTranslation){
+    var testTranslate = [];
+    var resultWithComas = '';
+    var resultWithoutComas = '';
+
+    for (i=0; i < textForTranslation.length; i++){
+        if (textForTranslation[i] == ''){
+            testTranslate.push(textForTranslation[i]);
+        }else if((textForTranslation[i] == 'a')||(textForTranslation[i] == 'e')||(textForTranslation[i] == 'i')||(textForTranslation[i] == 'o')||(textForTranslation[i] == 'u')){
+            testTranslate.push(textForTranslation[i]);
+        }else{
+            testTranslate.push(textForTranslation[i] + 'o' + textForTranslation[i]);
+        }
+    }
+    resultWithComas = testTranslate.toString();
+    resultTranslation = result.replace(/,/g, '');
+    return resultTranslation;
+}
+
+/*
+sum() & multiply()
+Define a function sum() and a function multiply() that sums and multiplies (respectively) all the numbers in an array of numbers. For example, sum([1,2,3,4]) should return 10, and multiply([1,2,3,4]) should return 24.
+*/
+
+function sum(arrayToSum){
+    var arraySum = arrayToSum;
+    var quantity = arraySum.length - 1;
```

Comment:
A more semantic variable would be `totalNumbers`

Reply:
Ok. Changed.